esmvalcore/_citation.py

```diff
@@ -0,0 +1,244 @@
+"""Citation module."""
+import logging
+import os
+import re
+import textwrap
+from functools import lru_cache
+
+import requests
+
+from ._config import DIAGNOSTICS_PATH
+
+logger = logging.getLogger(__name__)
+
+REFERENCES_PATH = DIAGNOSTICS_PATH / 'references'
+
+CMIP6_URL_STEM = 'https://cera-www.dkrz.de/WDCC/ui/cerasearch'
+
+# The technical overview paper should always be cited
+ESMVALTOOL_PAPER = (
```

Comment:
This is now published in final form, please update the reference to `righi20gmd`, see https://www.geosci-model-dev.net/13/1179/2020/.

Reply:
Done